tensornetwork/linalg/operations.py

```diff
@@ -343,4 +343,4 @@ def ncon(
                             out_order=out_order, check_network=check_network,
                             backend=backend)
   output = Tensor(res, backend=backend)
-  return output
+  return output
```

Comment:
what kind of weird character is there are the end of this line?

Reply:
I don't see a weird character

Reply:
<img width="1440" alt="Screen Shot 2020-07-15 at 8 54 25 PM" src="https://user-images.githubusercontent.com/20912915/87614194-5f781a00-c6dd-11ea-895b-5af9780ec3db.png">